qa/tasks/cephfs/caps_helper.py

```diff
@@ -46,8 +74,6 @@ def conduct_pos_test_for_read_caps(self, filepaths, filedata, mounts):
                     self.assertEqual(data, contents)
 
     def conduct_pos_test_for_write_caps(self, filepaths, mounts):
-        filedata = ('some new data on first fs', 'some new data on second fs')
```

Comment:
You remove the local `filedata` parameter in the first commit, but it's still used by the following code in this func. Maybe you have fixed in later commits, but it's hard to review for the code, could you fix it here ?

Reply:
Yes, that would be better. Fixed now.

Reply:
Moved here now - https://github.com/ceph/ceph/pull/46994/commits/370449ac9d088ba33111fb4948b1bf59f7901168